### PR TITLE
Sort selected frequencies by central wavelength

### PR DESCRIPTION
**Pull request by cquiroz**
Signed-off-by: Carlos Quiroz <cquiroz@gemini.edu>

### CODE REVIEW

File: model/shared/src/main/scala/explore/model/ImagingConfigurationOptions.scala [hunk 1]
```diff
@@ -24,16 +27,21 @@ sealed abstract class AvailableFilter {
   val range: Option[Quantity[Int, Nanometer]]
 }
 
+object AvailableFilter {
+  implicit val order: Order[AvailableFilter]       = Order.by(x => (x.centralWavelength, x.tag))
+  implicit val ordering: Ordering[AvailableFilter] = order.toOrdering
```
Comment (rpiaggio) [inline model/shared/src/main/scala/explore/model/ImagingConfigurationOptions.scala:32]: I'm almost sure that cats provides an implicit conversion from `Order` to `Ordering` if you import `cats.syntax.all._` (probably?) at the use site.

**Review by rpiaggio — APPROVED** [review]
LGTM

**Review by toddburnside — APPROVED** [review]
LGTM

File: model/shared/src/main/scala/explore/model/ImagingConfigurationOptions.scala (hunk 1 above)
Comment (cquiroz) [reply model/shared/src/main/scala/explore/model/ImagingConfigurationOptions.scala:32]: That's correct but you need to import `cats.implicits._` for that to work, Strange

**Pull request merged**